packages/libs/components/src/components/tidytree/TreeTable.tsx

```diff
@@ -74,18 +91,24 @@ export default function TreeTable<RowType>(props: TreeTableProps<RowType>) {
         {...props.treeProps}
         rowHeight={rowHeight}
         leafCount={rows.length}
-        options={{ margin: [0, 10, 0, 10] }}
+        options={{ margin: [0, 10, 0, 10], interactive: false }}
       />
-      <>
+      <div
+        style={{
+          flexGrow: 1,
+          width: 1 /* arbitrary non-zero width seems necessary for flex */,
+        }}
+      >
         <Global
           styles={globalStyle`
 	  .DataTable {
 	    margin-bottom: 0px !important;
+            width: 80%;
 	  }
 	`}
         />
         <Mesa state={tableState} />
-      </>
+      </div>
```

Comment:
Sorry, I'm just noticing this. I'm not sure we should use `Global` here, as the style will be reflected on all subsequent pages.

Instead, could you use a `css` prop on the parent `div` component (line 96), and target `.DataTable` in there?

```tsx
      <div
        css={{
          flexGrow: 1,
          width: 1 /* arbitrary non-zero width seems necessary for flex */,
          '.DataTable': {
            marginBottom: '0px !important',
            width: '80%'
          }
        }}
      >
        <Mesa state={tableState} />
    </div>

```

Reply:
Oh yeah, that's much simpler (and no side effects) - done in https://github.com/VEuPathDB/web-monorepo/pull/904/commits/5c896a0152eacb19703484cce39ce670734fc8c8